Model/Response/Transactions.php

```diff
@@ -39,8 +44,9 @@ public function process()
         parent::process();
 
         $body = $this->getBody();
-        $this->_token = $body['token'];
+        $this->_token = isset($body['token']) ? $body['token'] : '';
```

Comment:
is this the get transaction response? if so, token is always available

Reply:
This is called for both POST and GET method, so we have checked if token is set.

Reply:
it will be set on both

Reply:
actually we don't get token from GET call response.

Reply:
Why do we use the same logic for POST / GET?